cmd/bots/internal/telegram/handlers/handlers.go

```diff
@@ -154,6 +154,43 @@ func InitTgHandlers(environment *common.TelegramBotEnvironment, requestType chan
 			)
 		}
 		return RemoveNodeHandler(c, environment, requestType, responsePairType, args[0])
+
+	})
+
+	environment.Bot.Handle("/add_alias", func(c tele.Context) error {
+		args := c.Args()
+		if len(args) != 2 {
+			return c.Send(
+				messages.AliasWrongFormat,
+				&tele.SendOptions{
+					ParseMode: tele.ModeDefault,
+				},
+			)
+		}
+		return UpdateAliasHandler(c, environment, requestType, args[0], args[1])
+	})
+
+	environment.Bot.Handle("/aliases", func(c tele.Context) error {
+		nodes, err := messaging.RequestAllNodes(requestType, responsePairType)
+		if err != nil {
+			environment.Zap.Error("failed to get nodes list", zap.Error(err))
+		}
+		var msg string
+		for _, n := range nodes {
+			if n.Alias != "" {
+				msg += fmt.Sprintf("Node: %s\nAlias: %s\n\n", n.URL, n.Alias)
```

Comment:
Please, use `strings.Builder` + `fmt.Fprintf`.

Reply:
Okay...  I'm closing this conversation If you didn't find this comment useful...